src/System Application/App/Edit in Excel/src/EditinExcelImpl.Codeunit.al

```diff
@@ -865,13 +867,32 @@ codeunit 1482 "Edit in Excel Impl."
         if not TenantWebService.Get(TenantWebService."Object Type"::Page, ServiceName) then
             exit;
 
-        EditinExcelFilters.ReadFromJsonFilters(Filter, Payload, TenantWebService."Object ID");
+        EditinExcelFilters.ReadFromJsonFilters(Filter, Payload, TenantWebService."Object ID", FilterErrors);
         EditinExcel.OnEditInExcelWithFilters(ServiceName, EditinExcelFilters, SearchString, Handled);
         if Handled then begin
             Session.LogMessage('0000IG8', EditInExcelHandledTxt, Verbosity::Normal, DataClassification::SystemMetadata, TelemetryScope::ExtensionPublisher, 'Category', EditInExcelTelemetryCategoryTxt);
             exit;
         end;
-
+        if FilterErrors.Count() > 0 then
+            Message(StrSubstNo(EditInExcelInvalidFilterErr, FormatFilterErrors(FilterErrors)));
         GetEndPointAndCreateWorkbookWStructuredFilter(ServiceName, EditinExcelFilters, SearchString);
     end;
-}
+
+    local procedure FormatFilterErrors(FilterErrors: Dictionary of [Text, Boolean]): Text
+    var
+        ConcatenatedErrors: Text;
+        ErrorText: Text;
+        IsFirst: Boolean;
+    begin
+        ConcatenatedErrors := '';
+        IsFirst := true;
+
+        foreach ErrorText in FilterErrors.Keys() do begin
```

Comment:
I don't like the pattern used here.
Take a look how it already done in Base App for example:
![image](https://github.com/microsoft/BCApps/assets/59221826/af42246e-66b9-4591-bd40-6d4351ed76d7)

So, adding, than removing trailing ','